class_character.py

```diff
@@ -0,0 +1,83 @@
+import csv
+
+
+class Character:
+    max_health = 100
+
+    def __init__(self, occupation: str, first_name: str, last_name: str, age: int):
+
+        self.features = {}
+        with open(f"./CHARACTERS/{occupation}.txt", 'r') as file:
+            reader = csv.reader(file)
+            for row in reader:
+                if row:
+                    key, feature = row
+                    self.features[key] = feature
```

Comment:
Sugeruję zmodyfikować ten init i  napisanie go tak żeby możliwe było tworzenie instancji z różnych źródeł.  Napisanie tego w ten sposób niejako wiąże tą klasę z generowaniem jej instancji z pliku tekstowego.

Reply:
Mówiąc szczerze, nie bardzo wiem, jak to zrobić. To znaczy, podejrzewam, że powinienem zrobić funkcję poza klasą (żeby nie była w każdej klasie) która zwróci {self.features}, na której będą się budowały instancje klasy. Problem w tym, że nie wiem, jak zrobić funkcję, która zbierze dane z różnych źródeł (i co to będą te inne źródła poza plikiem tekstowym?).